### PR TITLE
Action plan on workshop-APIs: add explanation for code coverage

### PR DESCRIPTION
**Pull request by Heathercoraje**
docs for action points [here](https://github.com/foundersandcoders/master-reference/issues/630#issuecomment-331800134)

WS - API

AP: Longer workshop or rebalance the different sections of the workshop to allow more room for code coverage.
AP: TDD week should have a workshop/morning challenge regarding code coverage

### CODE REVIEW

File: README.md
Comment (emilyb7) [inline README.md]: typo? (coverge)

File: README.md
Comment (emilyb7) [inline README.md]: I don't know much about the difference between these NPM packages, but it sounds like they're quite similar? Would it be better to avoid confusion by sticking with one recommendation? In my experience, FAC people generally use istanbul...

**Review by emilyb7 — COMMENTED** [review]
Hey @Heathercoraje thanks for this.

I don't really know much about the context in which these changes have been proposed. What was the discussion in Nazareth? Have students in London also raised issues?

Happy for the changes to be merged in, but please assign more people (ideally from multiple campuses) as reviewers to make sure everyone's happy to teach this way :)

File: README.md
Comment (ameliejyc) [reply README.md]: I think initially we used istanbul but then switched to nyc as it has full ES6 support?

Comment (Heathercoraje) [reply README.md]: I thought nyc is a tool built on istanbul so that Istanbul and nyc are basically one set of code coverage tool now?
https://github.com/istanbuljs/nyc/issues/524
https://nodejs.libhunt.com/project/nyc/vs/istanbul

File: README.md
Comment (Heathercoraje) [reply README.md]: Sorry, somehow I completely missed the notification for this. 
I am sharing [action points](https://github.com/foundersandcoders/master-reference/issues/630#issuecomment-331800134) to be made in the meeting with London, Gaza and Naz Campus.

**Comment by mantagen** [comment]
@emilyb7 i'm not able to add any reviewers ... it might be that @Heathercoraje can't either.

The intended reviewers are @ameliejyc @rebecacalvoquintero @Walidmash @Eslam3bed @mantagen

It might be worth us adding it to foundersandcoders repo and you, @emilyb7 , being CODEOWNER [similar to this](https://github.com/foundersandcoders/ws-software-design-js).

**Review by mantagen — APPROVED** [review]

**Comment by emilyb7** [comment]
ok, i've added everyone tagged as a collaborator!

**Comment by Heathercoraje** [comment]
@emilyb7  I couldn't add @Walidmash

**Comment by emilyb7** [comment]
@Heathercoraje that's because he hasn't accepted the invitation to collaborate. There's nothing I can do about that!

**Comment by mantagen** [comment]
@emilyb7 are you okay for this to be merged now? If not what are changes you'd like to see before it gets merged?
thanks!

**Comment by emilyb7** [comment]
@mantagen @Heathercoraje 

I think it would be good (as per my original comments) if there were more clarity regarding this...
```
Nyc is Istanbul's state of the art command line interface. The original purpose of nyc was to allow a user to capture test coverage.
These two started separately but have been merged.
```
Test coverage is not really the subject of the workshop, so it would be better to give a clear recommendation and not to confuse people.

However, that's just my opinion.

I'm not the one teaching the workshop. I just want to make sure that everyone is happy!

File: README.md
Comment (mantagen) [reply README.md]: @Heathercoraje can we just get them to use `nyc` rather than giving them choice?
if you're okay with that could you make the change to this section and push it

Comment (mantagen) [reply README.md]: @emilyb7 I don't know where @Heathercoraje but i've made a change. I get what you mean about the Code Coverage thing being kinda out of place, but that was the outcome of the the curriculum planning session and if it doesn't work it will hopefully serve to improve that process.
Let know your thoughts
Thanks

Comment (emilyb7) [reply README.md]: @mantagen great thanks! I'm happy to merge as long as that's cool with the new process?

Comment (mantagen) [reply README.md]: merge at will

**Pull request merged**
